lgc/patch/PatchInOutImportExport.cpp

```diff
@@ -1024,11 +1027,6 @@ void PatchInOutImportExport::visitCallInst(CallInst &callInst) {
           patchMeshGenericOutputExport(output, loc, locOffset, elemIdx, vertexOrPrimitiveIdx, isPerPrimitive, builder);
           break;
         }
-        case ShaderStageFragment: {
-          assert(callInst.arg_size() == 3);
-          llvm_unreachable("Fragment shader export should have been handled by the LowerFragColorExport pass");
-          break;
-        }
```

Comment:
Seems like this is an unrelated change?

Reply:
Oh. I mistakenly changed this code. Should be in another irrelevant PR.